src/main/content/blog_tags.json

```diff
@@ -51,6 +51,10 @@
             "name": "Kubernetes",
             "posts": ["12-factor-microprofile-kubernetes"]
         }
```

Comment:
There is a missing comma after the kubernetes section

Reply:
Added